pkg/backend/openshiftcluster/openshiftcluster.go

```diff
@@ -44,7 +44,7 @@ type manager struct {
 
 	ocDynamicValidator validate.OpenShiftClusterDynamicValidator
 
-	groups         features.ResourceGroupsClient
+	resourceGroups features.ResourceGroupsClient
```

Comment:
rofl, evidently these fields aren't even used any more!  I'll open a follow-up to sort that out.